docs/hashes.toml

```diff
@@ -46,7 +46,7 @@ index = 1
 
 # Spec: ckb_testnet
 [ckb_testnet]
-spec_hash = "0xe0e6adeaadc8d93ea7e3103722fb9344a19a97971f694b090260e76abb4d1c2d"
+spec_hash = "0xdde420d56c10a06749e79701b152ed1f532f4bdf0696b5896dfd97fbc821e231"
```

Comment:
The original value in v0.43.0:

https://github.com/nervosnetwork/ckb/blob/bbbe5a58f7024f1157fba75f26ab4e2fdfee1e38/docs/hashes.toml#L47-L49